exercises/practice/dnd-character/tests/Tests.elm

```diff
@@ -0,0 +1,223 @@
+                    "generated character dexterity should be within range"
+                <|
+                    \character ->
+                        Expect.all [ Expect.atLeast 3, Expect.atMost 18 ] character.dexterity
+            , skip <|
+                fuzzWith { runs = 100, distribution = Test.noDistribution }
+                    (Fuzz.fromGenerator DndCharacter.character)
+                    "generated character constitution should be within range"
+                <|
+                    \character ->
+                        Expect.all [ Expect.atLeast 3, Expect.atMost 18 ] character.constitution
+            , skip <|
+                fuzzWith { runs = 100, distribution = Test.noDistribution }
+                    (Fuzz.fromGenerator DndCharacter.character)
+                    "generated character intelligence should be within range"
+                <|
+                    \character ->
+                        Expect.all [ Expect.atLeast 3, Expect.atMost 18 ] character.intelligence
+            , skip <|
+                fuzzWith { runs = 100, distribution = Test.noDistribution }
+                    (Fuzz.fromGenerator DndCharacter.character)
+                    "generated character wisdom should be within range"
+                <|
+                    \character ->
+                        Expect.all [ Expect.atLeast 3, Expect.atMost 18 ] character.wisdom
+            , skip <|
+                fuzzWith { runs = 100, distribution = Test.noDistribution }
+                    (Fuzz.fromGenerator DndCharacter.character)
+                    "generated character charisma should be within range"
+                <|
+                    \character ->
+                        Expect.all [ Expect.atLeast 3, Expect.atMost 18 ] character.charisma
+            , skip <|
+                fuzzWith { runs = 100, distribution = Test.noDistribution }
+                    (Fuzz.fromGenerator DndCharacter.character)
+                    "generated character hitpoints should be 10 plus the constitution modifier"
+                <|
+                    \character ->
+                        Expect.equal character.hitpoints (10 + DndCharacter.modifier character.constitution)
+            , skip <|
+                fuzzWith { runs = 100, distribution = Test.noDistribution }
+                    (Fuzz.fromGenerator DndCharacter.character)
+                    "generated character abilities are not all equal"
+                <|
+                    \character ->
+                        let
+                            uniqueAbilities { strength, dexterity, constitution, intelligence, wisdom, charisma } =
+                                [ strength, dexterity, constitution, intelligence, wisdom, charisma ]
+                                    |> Set.fromList
+                                    |> Set.size
+                        in
+                        Expect.atLeast 2 (uniqueAbilities character)
```

Comment:
This one is checking that not all abilities are copies of each other. The highest most probable state with all the same states are all 13s ($p=13.2$%), which can happen with probability $p^6 = 10^{-11}$, very unlikely, so 100 runs is fine.
I'm pretty sure I could require having 3 separate values in 100 runs, but the calculations become more complex, so I got lazy and played it safe :)

An error for a test of this type looks like
```
Given { charisma = 10, constitution = 10, dexterity = 10, hitpoints = 10, intelligence = 10, strength = 10, wisdom = 10 }

    1
    ╷
    │ Expect.atLeast
    ╵
    2
```